namespaces/live.cloud-platform.service.justice.gov.uk/cloud-platform-service-pod/resources/ecr.tf

```diff
@@ -0,0 +1,79 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=5.2.0"
+
+  # REQUIRED: Repository configuration
+  team_name = var.team_name
+  repo_name = var.namespace
+  namespace = var.namespace
+
+  # REQUIRED: OIDC providers to configure, either "github", "circleci", or both
+  oidc_providers = ["github"]
+
+  # REQUIRED: GitHub repositories that push to this container repository
+  github_repositories = ["cloud-platform-terraform-service-pod"]
```

Comment:
The repo is not there yet: https://github.com/ministryofjustice/cloud-platform-terraform-service-pod